Monika After Story/game/script-topics.rpy

```diff
@@ -585,20 +585,20 @@ label monika_sayori:
         menu:
             m "If you're comfortable, that is.{fast}"
             "Yes.":
-                m "You know how Sayori is really clumsy?"
-                m 4rksdlb "Well, she kind of messed up the whole hanging thing..."
-                m "You're supposed to jump from high enough that the rope snaps your neck, making it quick and painless."
-                m "But she just used a chair, meaning she kind of just left herself to slowly asphyxiate."
-                m 4rksdla "But a few seconds in, she must have changed her mind or something..."
-                m "Because she started clawing at the rope, trying to free herself."
+                m 4eka "You know how Sayori is really clumsy?"
+                m 2rksdlb "Well, she kind of messed up the whole hanging thing..."
+                m 4rksdla "You're supposed to jump from high enough that the rope snaps your neck, making it quick and painless."
+                m 4rksdld "But she just used a chair, meaning she kind of just left herself to slowly asphyxiate."
+                m 2eksdlc "But a few seconds in, she must have changed her mind or something..."
+                m 2dksdld"Because she started clawing at the rope, trying to free herself."
```

Comment:
```suggestion
                m 2dksdld" Because she started clawing at the rope, trying to free herself."
```

Reply:
lmao you sure about that fix?

Reply:
~~oops~~ It should be fine now. Multi made a fix for the fix.

Reply:
also generally don't need to suggest things that cause travis failures, we can see that already.